### DISCUSSION TITLE
Issue with V2.0 Online Validator Tool for Large Files

### DISCUSSION THREAD

**Discussion by siddesai1005** [CMSgov/hospital-price-transparency#114] [Q&A] [answered]
We are currently experiencing an issue with the V2.0 Online Validator tool. Specifically, we have a few files that are quite large (around 300,000 KB), and it appears that the tool is unable to validate these files, possibly due to their size.
 
Could you please provide a solution or workaround for this issue? We need to ensure that all our files, regardless of size, can be validated properly.

**Comment by RidenourK** [comment, marked as answer]
@siddesai1005. Thank you for your question.
 
Could you please provide a screenshot of, or additional details about, what you are encountering with these large files?

Additionally, as indicated in the [Hospital Price Transparency Validator FAQs](https://www.cms.gov/files/document/hospital-price-transparency-validator-faqs.pdf) on the “[Resources](https://www.cms.gov/priorities/key-initiatives/hospital-price-transparency/resources)” page of the CMS HPT website, if you need to test a large file, we recommend you use the [command-line interface (CLI) validator](https://github.com/CMSgov/hpt-validator-cli) instead of the online validator. The CLI is a downloaded tool that runs locally in the user’s terminal, as opposed to the online validator, which runs in a user’s web browser.
